include/onnxruntime/core/providers/cuda/cuda_context.h

```diff
@@ -21,7 +21,7 @@ struct CudaContext : public CustomOpContext {
   cublasHandle_t cublas_handle = {};
   OrtAllocator* deferred_cpu_allocator = {};
 
-  void Init(const OrtKernelContext& kernel_ctx) override {
```

Comment:
Would you please add some comments to this file so that our users could know what is this for?

Reply:
Good point, cmments added to explain its purpose.